virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -16,35 +16,38 @@ Cluster configuration procedures require `cluster-admin` privileges.
 [id="virt-getting-started-before-you-begin"]
 == Before you begin
 
-* Review the xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[installation requirements].
-* Review the xref:../virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc#virt-features-for-storage[storage features] required for cloning, snapshots, and live migration. See xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider] for details.
+* Review xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Preparing your cluster for {VirtProductName}].
```

Comment:
I might rephrase this to make it more like the other bullets:

```suggestion
* Learn how to xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[prepare your cluster for {VirtProductName}].
```

Reply:
Thanks but I might rephase this to make it stronger. I don't want them to "learn" how to prepare their cluster. I want them to do it! :-)